pubspec.yaml

```diff
@@ -1,17 +1,37 @@
 name: dynamic_color_theme
 description: Dynamically change your color theme, including light and dark mode. Persist the colors across restarts.
-version: 1.1.2
+version: 1.2.0
 homepage: 'https://github.com/DFreds/dynamic_color_theme'
 
+# The following line prevents the package from being accidentally published to
+# pub.dev using `pub publish`. This is preferred for private packages.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
+# The following defines the version and build number for your application.
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by an optional build number separated by a +.
+# Both the version and the builder number may be overridden in flutter
+# build by specifying --build-name and --build-number, respectively.
+# In Android, build-name is used as versionName while build-number used as versionCode.
+# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
+# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
+# Read more about iOS versioning at
+# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
+
+
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.2
   flutter_colorpicker: ^0.4.0
```

Comment:
This could probably be upgraded to 0.5.0

Reply:
So I looked through ColorPicker's issues and currently, 0.5.0 is having problems. It was crashing your plugin. Once I switched it back to 0.4.0 it started working just fine.

Reply:
Ah okay, let's just leave it then for now.